src/main/java/teammates/common/datatransfer/attributes/StudentProfileAttributes.java

```diff
@@ -25,7 +25,9 @@
     public String email;
     public String institute;
     public String nationality;
-    public String gender; // only accepts "male", "female" or "other"
+    public enum gender{
```

Comment:
Follow the proper conventions for an enum. The enum name itself should start with a capital letter (like a class name). The enum contents should be in all caps.